CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+## Unreleased
+
+- [#913](https://github.com/stripe/stripe-react-native/pull/913) BREAKING CHANGE: Changed props for the `<AddToWalletButton />` component. Instead of passing `cardHolderName`, `cardLastFour`, `cardDescription`, and `cardBrand` directly as props, you will instead pass a `cardDetails` prop, which is an object containing the following fields:
+  - `primaryAccountIdentifier`: The `wallet.primary_account_identifier` value from the issued card.
```

Comment:
Should we mention that it's optional only in the sense that it's nilable, but users should aways pass it through as-is (i.e. nil or non-nil)?

Reply:
Sure, that makes sense and I can add that to the docs :)